File: CyberGooseReviewV2/Entity/Product.cs
using System.ComponentModel.DataAnnotations;

namespace CyberGooseReviewV2.Entity
{
    public class Product
    {
        [Key]
        public int Id { get; set; }
        public int CategoryId { get; set; }
        public Category Category { get; set; }
        public int Year { get; set; }
        public string Country { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string? YouTubeLink { get; set; }
        public int UserRating { get; set; }
        public int CriticRating { get; set; }
        public int CommonRating { get; set; }
        public byte[]? ProductPicture { get; set; }
    }
}

File: CyberGooseReviewV2/ClientApp/src/app/product.ts
import { Category } from "./category";
import { SubCategory } from "./sub-category";

export interface Product {
  id: number;
  categoryId: number;
  category: Category;
  subCategories: SubCategory[];
  name: string;
  description: string;
  youTubeLink: string | null;
  userRating: number;
  criticRating: number;
  commonRating: number;
  productPicture: any;
  year: number;
  country: string;
}
